ChangeLog.md

```diff
@@ -1,5 +1,8 @@
 ## Changelog
 
+4.1.1
+* FIX: add proxy and fix logger types
```

Comment:
v4.1.1 also adds support for sound objects with critical flags and true (bypass do not disturb) - https://stackoverflow.com/questions/58983109/apns-critical-alert-is-treated-as-normal-notification

> Added support for critical notifications (2) (#71)
>  
> See 2nd sound reference in https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/generating_a_remote_notification#overview

Reply:
Thanks for the heads up